rust/cardano-chain-follower/src/metadata/cip36.rs

```diff
@@ -2,68 +2,22 @@
 
```

Comment:
I would have expected everything inside this `metadata` module to be obsoleted by the cardano types crate, or the rbac crate?